lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -445,3 +445,9 @@ export async function addReleaseNotes(
   }
   return output;
 }
+
+export function shouldSkipChangelogMd(repository: string): boolean {
+  const skippedRepositories = ['facebook/react-native'];
```

Comment:
I think this should be a module scope const, otherwise it's re-created on each call.